### ISSUE TITLE
Avoid overwriting files in page media upload

### ISSUE THREAD

**Issue by honzabilek4** [getgrav/grav-plugin-admin#1249]
Hi,
can we make the media upload field to prevent image overwriting, while uploading a file with already present name? I know that it's possible to add own file upload field and set `avoid_overwriting` to `true`, but why it's not by default on page media? Is there any reason why to not do that? I think that would be way more user friendly than overwriting already uploaded files.

**Comment by rhukster** [comment]
This default behavior change could have some impact for people's workflow.  I commonly replace a `header.jpg` with different header images for a page.  I'm sure others do this also and just assume that it would be replaced.  I don't think `avoid_overwriting` should be the default behavior.

I would consider adding this option as an admin config option that would let you set it at runtime.

**Comment by honzabilek4** [comment]
Config option as in file upload would be nice :) I mean, it is confusing for some less technical people. Another option is showing a modal with info that a file with the same name exists and will be overridden so you cannot unintentionally override your files.

**Comment by zoof** [comment]
Would it be possible to allow uploads with the same file name? To prevent erroneous overwrites, UI could have a pop-up confirmation.

I had incorrectly posted the question to https://github.com/getgrav/grav/issues/2036.

**Comment by honzabilek4** [comment]
Yeah, I still think that overriding a file with the same name and thus destroying the old one without any notice is generally a bad idea.
